src/test/resources/feature/SignUp.feature

```diff
@@ -0,0 +1,52 @@
+@Irina
+Feature: SignUp
+
+  Background:
+    Given "Home" is displayed
+    And user clicks on "Sign in"
+    And "Login" is displayed
+    And user clicks on "Email address field"
+    And user enters "lalalala@emailll.com"
+    And user clicks on "Create an account"
+    And "CREATE AN ACCOUNT" is displayed
+
+  Scenario: SignUp - successful
+    When user enters VALID data into mandatory fields:
```

Comment:
Why is necesarry to specify that data is VALID and fields are mandatory? You can specify this in Scenario title and make this step more generic one

Reply:
Done